applications/asset_tracker_v2/src/modules/location_module.c

```diff
@@ -268,6 +268,10 @@ static void search_start(void)
 			DATA_FETCH_TIMEOUT_NEIGHBORHOOD_SEARCH * MSEC_PER_SEC;
 	}
 
+	if (methods_index_gnss != -1) {
+		config.methods[methods_index_gnss].gnss.priority_mode = true;
```

Comment:
Not very familiar with ATv2 logics, but is this settable from the cloud plain like this?
Or would this priority_mode be enough to be Kconfig in location lib?

Reply:
We cannot override this at all currently from ATv2 cloud configuration, ATv2 Kconfig or loclib Kconfig.
Related to this comment:
https://github.com/nrfconnect/sdk-nrf/pull/9325#discussion_r1025595333
We had a discussion with @simensrostad on Friday and currently I don't expect this feature to be turned on for ATv2 in NCS 2.2.0. That's my guess based on what I've heard.

Generally about the configurability of certain loclib features, I think we should work on them after NCS 2.2.0 on a broader scope than just one specific item.

Reply:
About loclib features and their configurability/usage from api, I see it as feature specific what are needed on api, what not, for example ncellmeas search types (new GCI search typed) are making very much impact on runtime, based on situation. Based on this to be removed code here, this particular feature looked like that runtime configuration isn't needed and thus was wondering why this was put to api and not as a Kconfig item?

Reply:
> this particular feature looked like that runtime configuration isn't needed and thus was wondering why this was put to api and not as a Kconfig item?

I guess to follow the general design of having configurability in the API's config structures. Whether this is a good design as such is another thing but that's how other features exist in GNSS configuration, and I would expect them to be mainly compile-time configurable from application point of view.

Reply:
I don't have a personal opinion, but if @trantanen says we will align configurability after NCS 2.2.0 I am tempted to leave it like this now.

Reply:
OK for me as well, although if keeping as runtime config, mosh location command support is needed as stated [in](https://github.com/nrfconnect/sdk-nrf/pull/9325#discussion_r1029099454).
In addition to mosh tests, I'd enable this after the PR merge in mosh_dev's ATv2 to testing feedback, because CIs are running there as well for ATv2. ping @moksanen.